.travis.yml

```diff
@@ -25,6 +25,14 @@ matrix:
    env: ADDITIONAL_BUILDS="debian" LINKAGE=std
    before_script:
     - ./configure --CFLAGS="-std=c99" --CXXFLAGS="-std=c++98" --install-deps --disable-lz4-ext --disable-regex-ext --prefix="$PWD/dest"
+ - name: "Linux GCC s390x: +Debian packages +BuiltinRegex +Strict"
```

Comment:
Do we really need all of these? I suggest just adding one single s390 worker with no additional builds

Reply:
@edenhill We simply just followed the way existing jobs are running on amd64. Could you please suggest which jobs are mandatory and we should keep and which ones are optional and OK to discard?

Reply:
Just copy the last job, "Linux GCC: +devel .." and remove the "+code-cov" and "rdutcoverage.sh" calls.

Reply:
@edenhill Updated the `.travis.yml` file as per review comment. Please check.

Reply:
@edenhill Incorporated the review comments. Please check.